docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3772,6 +3772,11 @@ kafka:
   # CLI flag: -ingest-storage.kafka.producer-max-record-size-bytes
   [producer_max_record_size_bytes: <int> | default = 15983616]
 
+  # The maximum allowed for a read requests processed by an ingester to wait
+  # until strong read consistency is enforced. 0 to disable the timeout.
```

Comment:
Two small comments:

1. Line 3775. We should specify the maximum what. Time in seconds? Etc.
2. Line 3776. We should use active language here, i.e. "To disable the timeout, enter 0".